src/main/java/com/algorand/algosdk/util/BoxQueryEncoding.java

```diff
@@ -19,6 +19,10 @@ public static String encodeBox(Box b) {
         return encodeBytes(b.name);
     }
 
+    public static String encodeBoxDescriptor(BoxDescriptor b) {
```

Comment:
Is this method for the sake of completeness? Would a user need to encode a `BoxDescriptor?

Reply:
@algochoi Primarily, completeness.  Open to discussion - my 2c:
* The SDK can't _safely_ make assumptions about which models will/won't be used.
* For example, consider the following _valid_ (I _think_) scenario using `BoxDescriptor`:  An API user queries all box names for a given app, and then queries for box contents by app + box name.

Reply:
Yes, I think that example makes sense - resolving